packages/adminrouter/extra/src/lib/cache.lua

```diff
@@ -234,8 +234,24 @@ local function fetch_and_store_marathon_apps(auth_token)
           goto continue
        end
 
+       local do_rewrite_requrl = labels["DCOS_SERVICE_REWRITE_REQUEST_URLS"]
+       if not do_rewrite_requrl then
+          -- Label is absent, assume 'true' by default:
+          do_rewrite_requrl = true
+       elseif do_rewrite_requrl == 'no' or do_rewrite_requrl == 'false' or do_rewrite_requrl == '0' then
+          ngx.log(ngx.INFO, "DCOS_SERVICE_REWRITE_REQUEST_URLS for app '" .. appId .. "' set to 'false'")
```

Comment:
Assuming that this logs upon every service access attempt: don't you think that this might be a little too noisy?

Reply:
Please note that this is `cache.lua` file not the `service.lua`. The line is printed on every cache refresh, not service access attempt.

Reply:
Cool, that's convincing.